packages/trader/src/AppV2/Utils/__tests__/sort-symbols-utils.spec.ts

```diff
@@ -0,0 +1,58 @@
+
+describe('sortSymbols', () => {
+    it('should sort symbols correctly according to market order', () => {
+        const symbolsList = [
+            { symbol: 'BTCUSD', display_name: 'BTC/USD', market: 'cryptocurrency', submarket_display_name: 'Bitcoin' },
+            { symbol: 'EURUSD', display_name: 'EUR/USD', market: 'forex', submarket_display_name: 'Euro/USD' },
+            {
+                symbol: 'OTC_N225',
+                display_name: 'Japan 225',
+                market: 'indices',
+                submarket_display_name: 'Asian indices',
+            },
+            { symbol: 'GOLD', display_name: 'GOLD', market: 'commodities', submarket_display_name: 'Gold' },
+            {
+                symbol: '1HZ100V',
+                display_name: 'Volatility',
+                market: 'synthetic_index',
+                submarket_display_name: 'Derived',
+            },
+        ];
+        const sortedSymbols = sortSymbols(symbolsList as ActiveSymbols);
+        expect(sortedSymbols).toEqual([
+            {
+                symbol: '1HZ100V',
+                display_name: 'Volatility',
+                market: 'synthetic_index',
+                submarket_display_name: 'Derived',
+            },
+            { symbol: 'EURUSD', display_name: 'EUR/USD', market: 'forex', submarket_display_name: 'Euro/USD' },
+            {
+                symbol: 'OTC_N225',
+                display_name: 'Japan 225',
+                market: 'indices',
+                submarket_display_name: 'Asian indices',
+            },
+            { symbol: 'BTCUSD', display_name: 'BTC/USD', market: 'cryptocurrency', submarket_display_name: 'Bitcoin' },
+            { symbol: 'GOLD', display_name: 'GOLD', market: 'commodities', submarket_display_name: 'Gold' },
+        ]);
+    });
+    it('should handle symbols with same market correctly', () => {
+        const symbolsList = [
+            { symbol: 'EURUSD', display_name: 'EUR/USD', market: 'forex', submarket_display_name: 'Euro/USD' },
+            { symbol: 'GBPUSD', display_name: 'GBP/USD', market: 'forex', submarket_display_name: 'Pound/USD' },
+            { symbol: 'BTCUSD', display_name: 'BTC/USD', market: 'cryptocurrency', submarket_display_name: 'Bitcoin' },
+            { symbol: 'ETHUSD', display_name: 'ETH/USD', market: 'cryptocurrency', submarket_display_name: 'Ethereum' },
+        ];
+
+        const sortedSymbols = sortSymbols(symbolsList as ActiveSymbols);
+        expect(sortedSymbols).toEqual([
+            { symbol: 'EURUSD', display_name: 'EUR/USD', market: 'forex', submarket_display_name: 'Euro/USD' },
+            { symbol: 'GBPUSD', display_name: 'GBP/USD', market: 'forex', submarket_display_name: 'Pound/USD' },
+            { symbol: 'BTCUSD', display_name: 'BTC/USD', market: 'cryptocurrency', submarket_display_name: 'Bitcoin' },
+            { symbol: 'ETHUSD', display_name: 'ETH/USD', market: 'cryptocurrency', submarket_display_name: 'Ethereum' },
+        ]);
```

Comment:
```suggestion
        expect(sortedSymbols).toEqual(symbolsList);
```
2 arrays are exactly the same in this test case. perhaps, it's better to use an example which requires some kind of sorting instead? like putting the ones with `market: 'cryptocurrency'` first :)

Reply:
yeah i switched the orders to verify the test